contracts/CounterApp.sol

```diff
@@ -1,8 +1,7 @@
 pragma solidity ^0.4.24;
 
-import "@aragon/os/contracts/apps/AragonApp.sol";
-import "@aragon/os/contracts/lib/math/SafeMath.sol";
-
+import "@aragon/abis/os/contracts/apps/AragonApp.sol";
```

Comment:
Hmm interesting, I am really not a big fan of this—`abi` would not include the actual contract itself and this seems very weird.

It's ok for now, but we should remember to update this once we release `@aragon/os` without all the truffle baggage.

Reply:
Yes agree, probably `@aragon/abis` was a good exercise to know some of the things we would like to improve on next version `@aragon/os`

Reply:
I also agree on this. Besides, does `@aragon/os` have a meaningful impact on the overall installation time?

Reply:
At the moment, yes, because it includes all the truffle baggage.

We're only releasing `@aragon/os@5` because we need to remove all that weight (and it's a breaking change).